src/ledger/LedgerTxnAccountSQL.cpp

```diff
@@ -667,7 +667,8 @@ LedgerTxnRoot::Impl::bulkLoadAccounts(UnorderedSet<LedgerKey> const& keys) const
     {
         BulkLoadAccountsOperation op(mApp.getDatabase(), keys);
         return populateLoadedEntries(
```

Comment:
I feel like this function would benefit from and overload or default parameter, as to avoid the confusing `nullptr` passed around.

Reply:
I don't see any changes here

Reply:
Sorry! done now 😄